pingdom/pingdom.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import requests
+import json
+import argparse
+
+parser = argparse.ArgumentParser(description='Collects monitoring data from Pingdom.')
+parser.add_argument('-u', '--pingdom-user-name', help='The Pingdom User Name', required=True)
+parser.add_argument('-p', '--pingdom-password', help='The Pingdom Password', required=True)
+parser.add_argument('-a', '--pingdom-api-key', help='The Pingdom API-KEY', required=True)
+
+class Pingdom:
+    def __init__(self, api_key, user_name, password):
+        self.api_key = api_key,
+        self.user_name = user_name,
+        self.password = password
+
+    def handle_error(self, error_message):
+        sys.stderr.write("ERROR:|Pingdom| " + error_message)
+        sys.exit(1)
+
+    def call_api(self, api):
+        headers = {'App-Key': self.api_key[0]}
+
+        base_api = 'https://api.pingdom.com/api/2.0/' + api
+        response = requests.get(base_api, headers=headers, auth=requests.auth.HTTPBasicAuth(self.user_name[0], self.password))
+
+        if response.status_code == 200:
+            return response.json()
+
+        else:
+            self.handle_error("API [" + base_api + "] failed to execute with error code [" + str(response.status_code) + "].")
```

Comment:
`handle_error` calls sys.exit(1). So this return statement will never be executed. Is that what you want?